angularjs-portal-frame/src/main/webapp/css/buckyless/header.less

```diff
@@ -72,7 +72,7 @@ a#myuw-header img {
 }
 .bucky-nav {
   background:linear-gradient(
-    rgba(20,20,20, 0.8), 
+    rgba(20,20,20, 0.8),
     rgba(20,20,20, 0.8)
```

Comment:
I think there needs to be 2 if you're using a gradient, but if it's just a color you can take out linear-gradient and just put the rgba color. I can look at it when I go through and clean up the styles.